src/components/avatar-image.tsx

```diff
@@ -18,5 +19,9 @@ interface AvatarImageProps {
 export const AvatarImage: React.VoidFunctionComponent<AvatarImageProps> = ({
   src,
 }) => {
-  return <StyledImg src={src ?? defaultAvatarImg} />;
+  return (
+    <ImageContainer>
+      <StyledImg src={src ?? defaultAvatarImg} width={28} height={28} />
```

Comment:
```suggestion
      <StyledImg src={src ?? defaultAvatarImg} />
```

(if we have `styled.img`, which can contain `height: 28px; width: 28px;`

Reply:
Updated